src/usr/local/www/services_dhcpv6.php

```diff
@@ -800,8 +803,8 @@ function dhcpv6_apply_changes($dhcpdv6_enable_changed) {
 	'ddnsdomainprimary',
 	'DDNS Server IP',
 	$pconfig['ddnsdomainprimary'],
-	'V4'
-))->setHelp('Enter the primary domain name server IPv4 address for the dynamic domain name.');
+	'V6'
+))->setHelp('Enter the primary domain name server IPv6 address for the dynamic domain name.');
```

Comment:
If the validation test gets changed as suggested, change this to match.

Reply:
changed to "BOTH"